R/filter_by_feature.R

```diff
@@ -28,12 +28,19 @@ filter_by_feature <- function(fltr_features, fltr_condition, matrix, verbose = T
       operator <- gsub("[[:alnum:]]", "", fltr_condition[i])
       value <- gsub("[^0-9]", "", fltr_condition[i])
       filtercolumn <- which(colnames(matrix) == fltr_features[i])
+      if (verbose) {
+        cat(paste("filtering features: before if (length(filtercolumn) == 0)\n"))
+      }
```

Comment:
did you add this?

Reply:
yes, this is where test_data will fail.
I'll remove this.